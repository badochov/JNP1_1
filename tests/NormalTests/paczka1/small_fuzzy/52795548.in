?A842
aJ1sGhMOl5 S753 23,6
?S"53
?aJ1sGhMOl5
nt1gtIfH S767 46,1
PctfHsgSv S767 64,1
S767 A842 17,3

S767 A842 52,0
S767 A842 105,0
?PctfHsgSv
PejuvOU A842 60,1
?A842
?PctfHsgSv
JYGk S767 10,1
JYGk S767 1,6