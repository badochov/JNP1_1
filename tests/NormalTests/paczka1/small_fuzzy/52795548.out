A842 34,7
